app/build.gradle

```diff
@@ -48,4 +48,5 @@ dependencies {
     implementation 'com.jakewharton:butterknife:8.5.1'
     implementation 'id.zelory:compressor:2.1.0'
     annotationProcessor 'com.jakewharton:butterknife-compiler:8.5.1'
+    implementation 'com.madgag.spongycastle:core:1.58.0.0'
```

Comment:
Why are you adding this?

Reply:
iText is dependent on spongycastle library for encryption of password.

Please refer this: https://viralpatel.net/blogs/password-protect-pdf-itext-java/

Reply:
okay. cool 👍